.github/workflows/release.yml

```diff
@@ -115,7 +115,7 @@ jobs:
 
       - name: build release
         shell: bash
-        run: cargo build --all-features --release ${{ matrix.config.extraArgs }}
+        run: cargo build --release ${{ matrix.config.extraArgs }}
```

Comment:
Removed `--all-features` completely since I think we just want `default` features for release artifacts.